test/unit/services/draft_edition_updater_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'test_helper'
+
+class DraftEditionUpdaterTest < ActiveSupport::TestCase
```

Comment:
I feel as though it would be worthwhile having a unit test for 'cannot perform if edition is invalid'